### PR TITLE
Added JIP LN NVSE Plugin and 'Wasteland Defense.esp' to the masterlist.

### PR DESCRIPTION
**Pull request by ghost**
I created a common group to JIP as there are many mods depending on the plugin.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -5120,6 +5126,12 @@ plugins:
         itm: 6
         udr: 0
         nav: 0
+  - name: 'Wasteland Defense.esp'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/38049/?' ]
+    msg:
+      - type: say
+        content: 'Contains wild edits to records 00116185 and 00116187.'
```
Comment (MacSplody) [inline masterlist.yaml]: It's a bit vague, are they supposed to remove those two records?

Comment (MacSplody) [reply masterlist.yaml]: Or just a part of them?

Comment (ghost) [reply masterlist.yaml]: The two records should be removed, they're going to break any mod that would interact with Benny or the Tops.

Comment (MacSplody) [reply masterlist.yaml]: You could add that to the message as well.

>Contains wild edits to records 00116185 and 00116187. These two records should be removed, as they're going to break any mod that would interact with Benny or the Tops.

Comment (ghost) [reply masterlist.yaml]: Fair enough, I'll commit the changes in a few minutes.

Comment (MacSplody) [reply masterlist.yaml]: Thanks, It will save time in future, because someone is going to ask why otherwise.

**Comment by MacSplody** [comment]
Changes are good, commits are a bit 😕.
I'm not going to hold up the pull request over it, just wondering if you'd be able to tidy them up first.
Which git client are you using?

**Comment by ghost** [comment]
> Changes are good, commits are a bit 😕.
> I'm not going to hold up the pull request over it, just wondering if you'd be able to tidy them up first.
> Which git client are you using?

I'm using official Git for Windows. What would you like to be done to the commit messages?

**Comment by MacSplody** [comment]
> I'm using official Git for Windows. What would you like to be done to the commit messages?

Messages are fine, was going to suggest reordering and squashing so it would only have two commits. One for JIP LN NVSE Plugin and another for Wasteland Defense. 
But I don't think you can do that in Github for Windows.

You could always close this pull and open two new pulls with one for JIP LN NVSE Plugin and another for Wasteland Defense. Would you be okay with that?

**Comment by ghost** [comment]
Give me 10 minutes and it'll be fixed.

**Pull request closed without merging**